react.js

```diff
@@ -1,23 +1,49 @@
 // React Assessments
 
-// 1. Write a simple React component that simply prints "I am a component" to the screen. Be sure to include all necessary imports, exports, etc...
 
-// 2. Practice With Loops: In React, we often use the map function to iterate over an array held in state. To get more practice with this, write a vanilla javascript for loop that outputs the same thing as the map function below. 
+// 1. Write a simple React component that simply prints "I am a component" to the screen. Be sure to include all necessary imports, exports, etc...
+import React, { Component } from './App.js'
+
+export default class Print extends Component {
+  constructor(){
+    super()
+      this.class = {
+        string = 'I am a component'
+      }
+    render(){
+      return()
+        <div>
+          {this.class}
+        </div>
+      }
+    }
+  }
+        //After, in the App.js file, I would import Print from react.js (this file) and call <Print /> in the return
+
+// 2. Practice With Loops: In React, we often use the map function to iterate over an array held in state. To get more practice with this, write a vanilla javascript for loop that outputs the same thing as the map function below.
 
 
 var stuffArray = [42, "Arthur Dent", 1978, "Zaphod", "Vogon", "Marvin, the Paranoid Android"]
 
 stuffArray.map(function(el, i){
   console.log(el + " is at index: " + i)
 })
+  let newArr = () => {
```

Comment:
Two things here: 
1) Run this code and make sure it returns what you expect.
2) Do a bit of formatting. You want to be able to distinguish at a glance where the functions and blocks open and close as well as which lines are steps of the function vs which are lines of the loop. It takes practice but having that down will save you a bunch of debugging time.